frontend/components/ClubEditPage/InviteCard.tsx

```diff
@@ -84,6 +83,13 @@ export default function InviteCard({ club }: InviteCardProps): ReactElement {
       })
   }
 
+  const updatePermissions = (opt: MembershipRole) => {
+    setInviteRole(opt)
+    if (['Member', 'Officer', 'Owner'].includes(inviteTitle)) {
```

Comment:
@01jongmin This could be changed to `MEMBERSHIP_ROLES.map(role => role.label).includes(...)`.

What you have now is fine, but it is a good idea to use existing constants when possible so that if we ever add more roles in the future, nothing breaks.

Reply:
Feel free to merge this after this is fixed!

Reply:
MB, I'll fix it and commit it again.
Regarding the failing codecov/patch, is it okay to ignore the error and merge?

Reply:
Thanks! Yup, you can safely ignore that.

Ideally it would be nice to have code coverage for everything, but unit tests are a pain. 😅